### DISCUSSION TITLE
blog/2023/08/what-is-blendos/

### DISCUSSION THREAD

**Discussion by giscus** [Ast3risk-ops/blog#7] [Giscus]
# blog/2023/08/what-is-blendos/

The blendOS team’s response to Chris Titus Tech’s recent video.

https://blog.shinken.studio/blog/2023/08/what-is-blendos/

**Comment by bayazidbh** [comment]
I'll edit this comment once I finished watching the video during lunch, but for now, this is a nice and calm response. There is gong to be a lot of scepticism towards blendOS due to various reasons, warranted or unwarranted, but by just keep going you guys have been doing pretty good in proving yourself.

> This made me realize that we haven’t seen another Ubuntu, another Debian, in ages. It’s all just rices of another distro.

We kinda do, with NixOS. And there are independent distro every now and then. It's just that people don't really use them all that much.

I think what people really want is every single modification to be applicable to the grandaddy distro they use, with the enablement being a one-click toggle, with perfect stability and looks exactly the way they wanted it to be.

It's not that they want a new one, because if they do then they'd be experimenting with Nix/Guix, Gentoo, Solus, Clear, Void, Bedrock, and others I can't name right now. They just want all these niceties, without having to reinstall or truly experiment.

It kinda reminds me of Windows users' approach where they would complain at Microsoft but either continues to use Microsoft or expect someone else to create an .exe program or .bat script that fixes their specific issues so that they can keep using what they're using, without the status quo changing on anyone's side.

Edit: Sounds like an "LTT Wan Show Statement" moment to me, which is kinda funny given Chris' channel's name. But at least he seems to realize that he shouldn't be so quick to throw such remarks when he didn't actually know how it works behind the scene.

It's understandable, I had that moment myself, but having tested blendOS now, it's actually pretty convenient to have everything exported automatically and having a GUI to do all the container management operations.

**Reply by Ast3risk-ops** [reply]
NixOS is too complex for the average user, but you have a point.

**Reply by Ast3risk-ops** [reply]
I also *do* feel like gamersnexus correcting LTT.

**Reply by bayazidbh** [reply]
> NixOS is too complex for the average user

So does Arch. And much like Arch, there are people making them simpler, like [SnowflakeOS](https://snowflakeos.org/), that can work as a pathway towards pure usage of the granddaddy.

**Reply by Ast3risk-ops** [reply]
I suppose, though I don't think new users would gravitate to that sort of thing.
